index.html

```diff
@@ -150,5 +150,36 @@ <h3 class="framework-text">Skills</h3>
                 <li>Terminal</li>
             </ul>
         </section>
+        <section>
+            <p class="text-form">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </p>
+            <form>
+                <label for = "name"></label>
+                <input type="text" id="name" class="input-padron" required placeholder="Full name">
+                <hr>
+                
+                <label for="mail"></label>
+                <input type="email" id="mail" class="input-padron" required placeholder="email address">
+                <hr>
+    
+                <label for="msj"></label>
+                <textarea rows="4" cols="50" maxlength="500" id="msj" placeholder="Enter text here"></textarea>
```

Comment:
- [OPTIONAL] I will highly recommend you add `required` here as it will make a good user experience not to submit an empty text field.
  -  A user cannot submit the form if one or more of the fields is empty.